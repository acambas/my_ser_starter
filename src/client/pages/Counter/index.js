import React from 'react'
import Counter from './counter'

const CounterPage = () => (
  <div>
    <Counter />
  </div>
)

export default CounterPage
