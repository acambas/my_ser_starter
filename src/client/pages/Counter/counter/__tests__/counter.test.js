import Counter from '../index'
import React from 'react'
import renderer from 'react-test-renderer'

test('Link changes the class when hovered', () => {
  const component = renderer.create(<Counter increment={100} />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
