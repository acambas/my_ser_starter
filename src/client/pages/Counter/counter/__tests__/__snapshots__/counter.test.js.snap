// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div
  className="hello_world"
>
  <img
    src="test-file-stub"
  />
  <div
    className="stuff"
  >
    0
     : counter qwe
  </div>
</div>
`;
